### PR TITLE
🎨 Design updates

### LINKED ISSUE #144: Fix some design issues
image
We shouldn't see part of the next image

image

### LINKED ISSUE #156: Design changes
## Filter buttons

Make it consistent with the search bar, even level

image

## Add gradient to banner for resources
Gradient Code:
background-image: linear-gradient(70deg, #431C72 0%, #6A3E9E 97%);
image

### PR DESCRIPTION
**Pull request by n3a9**
- [x] Decrease size of partner images and center
- [x] Fix background image to height of window on home
- [x] Lock registration page background
- [x] Add gradient to resource banner
- [x] Update filter buttons to be more consistent with search bar
- [x] Move ResourceBreadcrumb to banner in ResourceDetail
- [x] Move navigation to the right
- [x] Decrease height of banner on resource search page
- [x] Fix carousel bug on home page
- [x] Refactor homepage css class names

Closes #156, closes #144

### CODE REVIEW

File: client/src/components/desktop/HomeDesktop.js [hunk 1]
```diff
@@ -1,34 +1,26 @@
...
     <Row className="home-block-2" type="flex" justify="center" align="middle">
-      <Col className="home-block-2__left" span={6}>
-        <Row
-          className="home-block-2__left__text"
-          type="flex"
-          justify="start"
-          align="middle"
-        >
-          <Col span={20}>
-            <Textfit mode="single">
```
Comment (abetco) [inline client/src/components/desktop/HomeDesktop.js]: why did we have the textfit in the first place?

File: client/src/css/Home.css [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
-  letter-spacing: 2px;
+  padding-left: 8%;
+  padding-bottom: 6%;
 }
 
 .welcome-text {
-  color: white;
+  color: #fff;
   font-weight: bold;
+  font-size: 60px !important;
```
Comment (abetco) [inline client/src/css/Home.css]: whats the purpose of important

Comment (abetco) [reply client/src/css/Home.css]: do we need it

File: client/src/components/desktop/HomeDesktop.js (hunk 1 above)
Comment (n3a9) [reply client/src/components/desktop/HomeDesktop.js]: Honestly I have no idea

File: client/src/css/Home.css (hunk 2 above)
Comment (n3a9) [reply client/src/css/Home.css]: i think so it like forces an overrride
its bad practice but im too lazy to fix it

**Pull request merged**